spring-cloud-kubernetes-integration-tests/discovery/discovery-client-v2/pom.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I would like all integration tests going forward to use kind and not rely on fabric8.  See some of the other examples
https://github.com/spring-cloud/spring-cloud-kubernetes/tree/master/spring-cloud-kubernetes-integration-tests/spring-cloud-kubernetes-core-k8s-client-it

Reply:
i tried the new integration module locally but it complaining on cannot find spring-cloud related images from dockerhub, is the image private access?

Reply:
Shouldn't be.  All the images should get built as part of the build.  Which images is it complaining about?

Reply:
> com.github.dockerjava.api.exception.NotFoundException: Status 404: {"message":"No such image: springcloud/spring-cloud-kubernetes-client-config-it:2.0.0-SNAPSHOT"}


i got this error when running the test locally, can you elaborate how to build the image?

Reply:
Ah it is not part of the parent pom that is why.  This script should run and build everything 
https://github.com/spring-cloud/spring-cloud-kubernetes/blob/master/spring-cloud-kubernetes-integration-tests/run.sh